dex/testing/eth/harness.sh

```diff
@@ -183,9 +186,7 @@ echo "Sending 5000 eth to delta and gamma."
 
 # Our transactions will PROBABLY appear in one of these blocks.
 echo "Mining some blocks"
-"${NODES_ROOT}/harness-ctl/mine-beta" "1"
-"${NODES_ROOT}/harness-ctl/mine-alpha" "1"
-"${NODES_ROOT}/harness-ctl/mine-beta" "5"
+"${NODES_ROOT}/harness-ctl/mine-beta" "15"
```

Comment:
I had a few times where it looked like mining on alpha made transaction inclusion speed worse. Will attempt to solve the problem in a separate pr.

Reply:
Dang what a drag. Linking back to the [original discussion](https://github.com/decred/dcrdex/pull/1005#discussion_r637244211) and karalabe's [comment in a different thread.](https://github.com/ethereum/go-ethereum/issues/2769#issuecomment-230249342)

Reply:
Another related idea was to have the non-miners (clients) connected to each other or at least another non-miner (or not the primary miner): https://github.com/ethereum/go-ethereum/issues/2822#issuecomment-240998150

> ivooz: The problem goes away when all the non-mining nodes are connected to each other, previously they only knew about miner nodes.

> karalabe: ... your first topology were that all nodes were connected only to the miners. This can be problematic as the miners are heavily loaded churning out blocks, so if everyone is bugging them to distribute the data too, I can imagine that they are simply overwhelmed (you essentially DDOS the miners).
> 
> I'd recommend enabling discovery (with a custom bootnode, maybe one of the miners, should be fine) so nodes can properly connect to each other and form a decent overlay network to propagate the data.

Reply:
Oh and about the first idea of starting 2 miners, one guy said he "updated the file "static-nodes.json" accordingly".  Know what we should do there?

Reply:
Will look into this directly after I get #1019 up to speed.

Reply:
Yeah, still scratching my head. These convos are from 2016, so maybe something different. Also, putting sleeps in the wrong places in the harness can just make everything fail.

Reply:
Pretty sure I found and fixed the problem in #1130 Calling `miner.start()` `miner.stop()` in quick succession seems to mess with the ability of txs to make it into blocks.